docs/experiments-ciral-v1.0.md

```diff
@@ -0,0 +1,139 @@
+
+## Corpus Download
+
+### 1. Manual Download
+
+The CIRAL Corpus can be downloaded from the [Hugging Face Repo](https://huggingface.co/datasets/CIRAL/ciral-corpus).
+
+We can now index the documents for each language as a `MrTyDiCollection`  using Anserini bindings from Pyserini.
+
+```bash
+python -m pyserini.index.lucene --collection MrTyDiCollection \
+    --input path/to/hausa_passages.jsonl --index lucene-index.ciral-v1.0-ha \
+    --generator DefaultLuceneDocumentGenerator \
+    --language ha --pretokenized \
+    --threads 16 --storePositions --storeDocvectors --storeRaw \
+    >& logs/log.ciral-v1.0-ha &
+
+python -m pyserini.index.lucene --collection MrTyDiCollection \
+    --input path/to/somali_passages.jsonl --index lucene-index.ciral-v1.0-so \
+    --generator DefaultLuceneDocumentGenerator \
+    --language so --pretokenized \
+    --threads 16 --storePositions --storeDocvectors --storeRaw \
+    >& logs/log.ciral-v1.0-so &
+
+python -m pyserini.index.lucene --collection MrTyDiCollection \
+    --input path/to/swahili_passages.jsonl --index lucene-index.ciral-v1.0-sw \
+    --generator DefaultLuceneDocumentGenerator \
+    --language sw --pretokenized \
+    --threads 16 --storePositions --storeDocvectors --storeRaw \
+    >& logs/log.ciral-v1.0-sw &
+
+python -m pyserini.index.lucene --collection MrTyDiCollection \
+    --input path/to/yoruba_passages.jsonl --index lucene-index.ciral-v1.0-yo \
+    --generator DefaultLuceneDocumentGenerator \
+    --language yo --pretokenized \
+    --threads 16 --storePositions --storeDocvectors --storeRaw \
+    >& logs/log.ciral-v1.0-yo &
+```
+
+
+### 2.  Download Pre-Built Sparse Indexes (for BM25)
+
+As an alternative to running the build commands above, the BM25 prebuilt indexes for each language are also provided and can be downloaded below.
+
+- [Hausa](https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.ciral-v1.0-ha.20230721.e850ea.tar.gz)
+- [Somali](https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.ciral-v1.0-so.20230721.e850ea.tar.gz)
+- [Swahili](https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.ciral-v1.0-sw.20230721.e850ea.tar.gz)
+- [Yoruba](https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.ciral-v1.0-yo.20230721.e850ea.tar.gz)
+
+
+## Data Prep
+
+We would need the topic and qrel files from the [Hugging Face repo](https://huggingface.co/datasets/CIRAL/ciral) to reproduce the baselines.  
+
+Clone the repo and copy the files to `tools/topics-and-qrels` in the cloned `Pyserini` repo. This requires the Pyserini [dev install](https://github.com/castorini/pyserini/blob/master/docs/installation.md#development-installation).
+
+```bash
+git clone https://huggingface.co/datasets/CIRAL/ciral
+cp -r ciral/*/*/* $PYSERINI_PATH/tools/topics-and-qrels/
+```
+
+
+
+## Retrieval: Dev Topics
+
+```bash
+python -m pyserini.search.lucene --index ciral-v1.0-ha \
+    --topics tools/topics-and-qrels/topics.ciral-v1.0-ha-train \
```

Comment:
we hvn't add the topics to the tools/ yet?

Reply:
also could we name it `dev` instead of `train`? that'd be more accurate

Reply:
our just call it "sample"?